data_structures/src/chain.rs

```diff
@@ -2465,6 +2598,27 @@ pub fn generate_unspent_outputs_pool(
     unspent_outputs
 }
 
+/// Method to sort own_utxos by value
+pub fn sort_own_utxos(
```

Comment:
This can be a method of `OwnUnspentOutputsPool` so in the future we can optimize the implementation by adding a sorted index or a cache.